tools/install-promethues-server.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+TOOLS_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
+
+pushd $TOOLS_DIR
+
+# Download Prometheus server.
+if [[ "$OSTYPE" == "linux"* ]]; then
```

Comment:
We've been doing it like

https://github.com/ray-project/ray/blob/32bf23d24fdb635ea781eb94fdebb8139d7a4009/ci/travis/install-bazel.sh#L6-L17

in the rest of the codebase.

Reply:
But how do I reuse this in my shell script?
write it as a function or copy it into my script?

Reply:
Just copy it into the script so that it is the same.

Reply:
Please don't forget about this comment.